convex/notebooks.ts

```
@@ -491,6 +625,42 @@ export const getByUrlWithPassword = query({
   },
 });
 
+// Server-side session token validation
+const validateSessionToken = async (ctx: any, sessionToken: string, notebookId: Id<"notebooks">) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)